google/cloud/aiplatform/training_jobs.py

```diff
@@ -1620,9 +1760,29 @@ def run(
         Any of ``training_fraction_split``, ``validation_fraction_split`` and
```

Comment:
I would add a line above this `Data fraction splits` block that states:

```
If training on a Verex AI dataset you can use one of the following split configurations: 
```

Apply this comment on all `run` docstrings.

Reply:
Done